docs/source/intervention_models/iv_iron_postpartum/index.rst

```diff
@@ -83,138 +70,46 @@ Vivarium Modeling Strategy
     - Affected measure
     - Effect size measure
     - Effect size
-    - Note
-  * - 
-    - 
-    - 
-    - 
-    - 
-    - 
+    - Note 
+  * - Hemoglobin
+    - Modelable entity
+    - 10487
+    - Population mean hemoglobin concentration (as continuous measure)
+    - Mean difference
+    - +10 g/L
     - 
 
 Hemoglobin level among pregnant and lactating women
 +++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-.. todo::
-
-  Link to existing document of the affected outcome (ex: cause or risk exposure model document)
-
-.. todo::
-
-  Describe exactly what measure the intervention will affect
-
-.. todo::
-
-  Fill out the tables below
-
-.. list-table:: Maternal hemoglobin restrictions
-  :widths: 15 15 15
-  :header-rows: 1
-
-  * - Restriction
-    - Value
-    - Note
-  * - Male only
-    - 
-    - 
-  * - Female only
-    - 
-    - 
-  * - Age group start
-    - 
-    - 
-  * - Age group end
-    - 
-    - 
-  * - Other
-    - 
-    - 
-
 .. list-table:: Maternal hemoglobin effect size
   :widths: 15 15 15 
   :header-rows: 1
 
   * - Population
     - Effect size
     - Note
-  * - 
-    - 
-    - 
-.. todo::
-
-  Describe exactly *how* to apply the effect sizes to the affected measures documented above
-
-.. todo::
+  * - Immediately post-birth women with hemoglobin levels less than 100 g/L
+    - +10 g/L 
+    - Effect is present two weeks after birth through the end of the postpartum period. Effect size is from the BMGF optomistic target product profile 
 
-  Note research considerations related to generalizability of the effect sizes listed above as well as the strength of the causal criteria, as discussed on the :ref:`general research consideration document <general_research>`.
+The effect of the IV iron intervention on maternal hemoglobin should be applied as an additive shift to the simulant's continuous hemoglobin exposure *two weeks following birth birth*. The effect of the intervention should be removed (subtracted from the hemoglobin exposure level) at the end of the postpartum period prior to the removal of the pregnancy adjustment factor.
 
 Assumptions and Limitations
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Validation and Verification Criteria
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Infant birthweight
-+++++++++++++++++++++++++++++++++++++++++++++++++++++
-
-.. todo::
-
-  Link to existing document of the affected outcome (ex: cause or risk exposure model document)
-
-.. todo::
-
-  Describe exactly what measure the intervention will affect
+- We assume the effect of the intervntion persists through the end of the postpartum period at which point the woman's hemoglobin concentration returns to its pre-pregnancy level. In reality, it may persist beyond the end of the postpartum period, in which case we may underestimate the intervention impact.
```

Comment:
```suggestion
- We assume the effect of the intervention persists through the end of the postpartum period at which point the woman's hemoglobin concentration returns to its pre-pregnancy level. In reality, it may persist beyond the end of the postpartum period, in which case we may underestimate the intervention impact.
```